app/views/authors/index.html.erb

```diff
@@ -0,0 +1,61 @@
+<%= stylesheet_link_tag 'authors', media: 'all', 'data-turbolinks-track': 'reload' %>
+
+<p id="notice"><%= notice %></p>
+
+<p id="authors">Authors</p><br>
+
+<%= form_tag authors_path, { :method => 'get', class: "container", role: "search" }  do %>
+	<div class="row">
+        <div class="col-md-6">
+					<h4>Author</h4>
+            <div id="custom-search-input">
+                <div class="input-group col-md-12">
+                  <div class="form-group">
+                     <%=text_field_tag :search,"",class:"form-control",placeholder:"Author" %>
+                  </div>
+                    <span class="input-group-btn">
+                        <button class="btn btn-info btn-lg" type="button">
+                            <i class="glyphicon glyphicon-search"></i>
+                        </button>
+                    </span>
+                </div>
+            </div>
+        </div>
+	</div>
+<% end %>
+
+<br><br><br>
+
+<div style="margin-left: 1162px">
```

Comment:
Avoid inline CSS: these properties should be in classes or IDs defined in your .scss files